pkg/controller/kubelet-config/kubelet_config_controller.go

```diff
@@ -364,10 +409,24 @@ func (ctrl *Controller) syncKubeletConfig(key string) error {
 		return ctrl.syncStatusOnly(cfg, err)
 	}
 
+	features, err := ctrl.featLister.Get(clusterFeatureInstanceName)
+	if err != nil && !errors.IsNotFound(err) {
+		err := fmt.Errorf("Could not fetch FeatureGates: %v", err)
```

Comment:
first char should be lower case in error message

Reply:
fixed. thanks.